pkgs/applications/networking/sniffers/sngrep/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, autoconf, automake, fetchFromGitHub, libpcap, ncurses, openssl, pcre }:
+
+stdenv.mkDerivation rec {
+  pname = "sngrep";
+  version = "1.4.6";
+
+  src = fetchFromGitHub {
+    owner = "irontec";
+    repo = "${pname}";
```

Comment:
`repo = pname;` is sufficient as `pname` is a string already. "${pname}" effectively converts pname from a string to a string and places inside an empty string